OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -880,16 +921,19 @@ public IEnumerable<IOrderTargeter> Orders
 		{
 			get
 			{
+				yield return new EnterAlliedActorTargeter<BuildingInfo>("ForceEnter", 6,
```

Comment:
It would in principle be cleaner if `ForceEnter` was implemented via a flag in the `ExtraData` of the `EnterOrder`, and likewise for `Land` on `Move`. I'm not sure how to cleanly implement the order targeter side of this, so this consider this a wishful thought rather than a request.

Reply:
I agree, but I don't know how to do this either so I'll leave it like this.